docs/authentication/saml/jit-provisioning.mdx

```diff
@@ -0,0 +1,17 @@
+---
+description: Learn about Clerk's support for Just-In-Time account provisioning with all supported SAML providers.
```

Comment:
I would say let's stick with the term `Just-in-Time (JIT) Provisioning` for the rest of the page, which is the most common one

```suggestion
description: Learn about Clerk's support for Just-in-Time (JIT) Provisioning during SAML SSO.
```